scipy/stats/_hypotests.py

```diff
@@ -139,6 +141,253 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
     return Epps_Singleton_2sampResult(w, p)
 
 
+PoissonMeansTestResult = make_dataclass('PoissonMeansTestResult',
+                                        ('statistic', 'pvalue'))
+
+
+def poisson_means_test(count1, nobs1, count2, nobs2, diff=0,
+                       alternative='two-sided'):
+    r"""
+    Calculates the poisson mean test, the "e-test", for the mean difference of
```

Comment:
```suggestion
    Calculates the poisson mean test, the "E-test", for the mean difference of
```

in future uses it is capitalized, we should be consistent here with upper case or change the other uses to lower case